pkg/routeagent-driver/cni-driver/cni-iface.go

```diff
@@ -0,0 +1,67 @@
+package cni_driver
```

Comment:
Since we are doing refactors, some suggestions, but take it only as opinion.

I would name this module cni_interface (instead of cni-driver)

Also I believe we always use the "_" convention instead of "-" for modules because it's problematic for go... (the import names don't translate directly)

Reply:
It's the file names, then when you do an import I believe golang will need a cni_drive "pkg/routeagent-driver/cni-driver" , at least on my side GoLand editor was getting crazy otherwise   

But I'm not sure now. Not important anyway